documentation/modules/mtv-changelog-2-7.adoc

```diff
@@ -0,0 +1,1356 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Release_notes/master.adoc
+
+:_content-type: PROCEDURE
+[id="mtv-changelog-2-7_{context}"]
+= {project-short} changelog
+
+The following changelog for {project-first} is a full list of packages used in the {project-short} 2.7 releases.
```

Comment:
This will require minimal maintenance, as thanks to @mnecas for providing the script it is a simple job to pull all the packages per version